mathml/presentation-markup/fractions/frac-parameters-1.html

```diff
@@ -62,7 +62,8 @@
 
   setup({ explicit_done: true });
   window.addEventListener("load", function() {
-    document.fonts.ready.then(runTests);
+    // Delay the check to workaround WebKit's bug https://webkit.org/b/174030.
+    setTimeout(() => { document.fonts.ready.then(runTests); }, 0);
```

Comment:
I think it's usually preferred `window.requestAnimationFrame()` instead of `setTimeout()`.
Could you try if something like that would fix the issue? Thanks!

Reply:
The WebKit bug is that `document.fonts.ready` resolves immediately, even when I replace the CSS @font-face with a list of document.fonts.add(...) just before document.fonts.ready. Using setTimeout or  requestAnimationFrame will delay things a bit and hence solve the issue, but I'm not sure requestAnimationFrame is more appropriate here, since it's not really a layout/painting issue but a font API bug. For example the following changes do *not* allow to workaround the issue:

    document.fonts.ready.then(() => { requestAnimationFrame(runTests); });
or 
   document.fonts.ready.then(() => { setTimeout(runTests, notSmallEnoughDelta); });

Reply:
I was thinking in something like:
```js
    window.requestAnimationFrame(() => { document.fonts.ready.then(runTests); });
```

However I agree with you that `setTimeout()` might be clearer, so I'm fine with keeping it.

Reply:
Yes, `requestAnimationFrame(() => { document.fonts.ready.then(runTests); });` would work too.